packages/react-components/src/components/styles/RichTextEditor.styles.ts

```diff
@@ -288,12 +285,9 @@ export const insertTableMenuTablePane = mergeStyles({
  */
 export const insertTableMenuFocusZone = (theme: Theme): string => {
   return mergeStyles({
-    lineHeight: '12px',
-    // fixed width is required to show columns in a grid correctly
-    width: '5rem',
```

Comment:
Just wanted  to confirm that the changes were tested in sample app and storybook

Reply:
Tested in both